servant-js/src/Servant/JS/Internal.hs

```diff
@@ -179,8 +179,8 @@ paramToStr qarg notTheEnd =
            <> "=' + encodeURIComponent("
            <> name
            <> if notTheEnd then ") + '" else ")"
-    Flag   -> name <> "="
-    List   -> name
+    Flag -> name <> "="
+    Servant.Foreign.List -> name
```

Comment:
This change doesn't seem like an improvement to me. Asymmetrically qualifying native (as in servant-provided) names just because `Lens` exports too many entities?